opm/simulators/utils/ParallelRestart.cpp

```diff
@@ -31,35 +31,11 @@
 #include <dune/common/parallel/mpitraits.hh>
 #if HAVE_MPI
 #include <ebos/eclmpiserializer.hh>
-#endif
```

Comment:
I'm probably blind, but I didn't see this `#endif` reinstated somewhere else.  Is that intentional?

Reply:
no, rebase meh. fixed.